infra/scripts/apply.sh

```
@@ -0,0 +1,86 @@
+      ;;
+  esac
+done
+
+echo "=== Multi-Cloud Deploy: OpenLegislation ==="
+echo "Environment: $ENV"
+echo "Provider: $PROVIDER"
+echo "Dry-run: $DRY_RUN"
+
+# Load env-specific vars (assume terraform.tfvars.$ENV)
+if [ -f "infra/terraform/terraform.tfvars.$ENV" ]; then
+  export $(grep -v '^#' infra/terraform/terraform.tfvars.$ENV | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)